tests/GraphBLAS-sharp.Tests/Common.fs

```diff
@@ -0,0 +1,34 @@
+namespace GraphBLAS.FSharp.Tests
+
+open System
+
+type VectorType =
+    | Sparse = 0
```

Comment:
Why enums?

Reply:
Because they are easier to enumerate. They could be enumerated without reflections.